config/core/resources/subscription.yaml

```diff
@@ -41,24 +41,12 @@ spec:
                   apiVersion:
                     description: API version of the referent.
                     type: string
-                  fieldPath:
```

Comment:
jotting here, but L38 still has ObjectReference, that description should also be updated.